spec/fixtures/http.ts

```diff
@@ -39,15 +39,15 @@ export function mockRCVariableFetch(
   data: any,
   token: string = 'thetoken'
 ): nock.Scope {
-  let mock: nock.Scope = nock('https://runtimeconfig.googleapis.com').get(
+  let interceptor = nock('https://runtimeconfig.googleapis.com').get(
```

Comment:
See https://github.com/firebase/firebase-functions/pull/450/files#r290017363

As for the annotation, it's no longer necessary as the inferred type is now correct.

Reply:
Hm, that link just takes me to the changed files section without selecting a particular file - which one should I be looking at?

Reply:
Hm, I guess GitHub broke something :) Here's a link to the same comment in the conversation view: https://github.com/firebase/firebase-functions/pull/450#discussion_r290017363

Just in case that one doesn't work either, here's a copy-paste:

> I had to do a small refactoring in this file so it compiles, since an updated version of nock ships with more strict (more correct) type definitions.
> 
> Renames were not strictly necessary. They follow names of interfaces defined by nock and I tried to make it more precise and descriptive. I can extract renames into a separate PR if you feel like it's too much changes to comfortably review and approve.

Reply:
Got it, thanks! This looks fine to me.